### PR TITLE
Migrated to Swift Package

### PR DESCRIPTION
**Pull request by shrtlist**
Converted to a Package so it can be used as a dependency with the Swift Package Manager.

### CODE REVIEW

File: Tests/SkopelosTests/SkopelosTests.swift [hunk 1]
```diff
@@ -0,0 +1,11 @@
+    import XCTest
+    @testable import Skopelos
+
+    final class SkopelosTests: XCTestCase {
+        func testExample() {
+            // This is an example of a functional test case.
+            // Use XCTAssert and related functions to verify your tests produce the correct
+            // results.
+            XCTAssertEqual(Skopelos().text, "Hello, World!")
```
Comment (albertodebortoli) [inline Tests/SkopelosTests/SkopelosTests.swift]: Thanks for contributing @shrtlist 🙏
The rest looks ok, if you could only address this.

**Review by albertodebortoli — CHANGES_REQUESTED** [review]

File: Tests/SkopelosTests/SkopelosTests.swift (hunk 1 above)
Comment (shrtlist) [reply Tests/SkopelosTests/SkopelosTests.swift]: You're welcome! Updated.

**Comment by shrtlist** [comment]
Any other thoughts about this PR, @albertodebortoli ?

**Review by albertodebortoli — COMMENTED** [review]
Sorry for the late reply @shrtlist all good, I'll re-open it and merge it.

**Comment by albertodebortoli** [comment]
Let's ignore Travis. Something to sort for another day.

**Pull request merged**